tree/dataframe/inc/ROOT/RDF/RTreeColumnReader.hxx

```diff
@@ -108,6 +110,23 @@ private:
    void *GetImpl(Long64_t entry) override;
 };
 
+class R__CLING_PTRCHECK(off) RMaskedColumnReader : public ROOT::Detail::RDF::RColumnReaderBase {
+   std::unique_ptr<ROOT::Detail::RDF::RColumnReaderBase> fValueReader;
+   std::unique_ptr<TTreeReaderValue<uint64_t>> fTreeValueMask;
+   unsigned int fMaskIndex = 0;
+
+   void *GetImpl(Long64_t) override;
+
+public:
+   RMaskedColumnReader(TTreeReader &r, std::unique_ptr<ROOT::Detail::RDF::RColumnReaderBase> valueReader,
+                       std::string_view maskName, unsigned int maskIndex);
+   virtual ~RMaskedColumnReader();
```

Comment:
Is this needed?

Reply:
In the .cxx, it's `= default`.

Reply:
Rule of five states that a class which declares (which includes `=default` and `=delete`) anyone of destructor, copy constructor, move constructor, copy assignment, move assignment should also declare all others. Again, that's a guideline, so I won't block this PR because the rule is not implemented.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all